src/test/java/gov/nih/nci/evs/api/controller/TermSuggestionFormControllerTests.java

```diff
@@ -262,14 +247,110 @@ public void testSubmitFormThrowsExceptionWhenSendEmailFails() throws Exception {
+    // SET UP
+    final String formPath = "formSamples/submissionFormTest.json";
+    final JsonNode formData = createForm(formPath);
+    // Create expected EmailDetails
+
+    // Mock the RecaptchaService to always return true for verifyRecaptcha
+    when(captchaService.verifyRecaptcha(anyString())).thenReturn(false);
+    doNothing().when(termFormService).sendEmail(any(EmailDetails.class));
+
+    // ACT & ASSERT
+    try {
+      termSuggestionFormController.submitForm(formData, null, recaptchaToken);
+      fail("Expected a ResponseStatusException to be thrown");
+    } catch (ResponseStatusException e) {
+      assertEquals(HttpStatus.BAD_REQUEST, e.getStatus());
+    }
+  }
+
+  /**
+   * Integration test to verify we can call the api path, path the formType, and return the JsonNode
+   * form
+   *
+   * @throws Exception exception
+   */
+  @Test
+  public void integrationTestGetFormTemplate() throws Exception {
+    // SET UP
+    final String formType = "ncit-form";
+    final String url = baseUrl + formType;
+    JsonNode form;
+
+    // ACT
+    log.info("Testing url: {}", url);
+    final MvcResult mvcResult =
+        this.mvc.perform(MockMvcRequestBuilders.get(url)).andExpect(status().isOk()).andReturn();
+    form = objectMapper.readTree(mvcResult.getResponse().getContentAsString());
+    log.info("Form = {}", form);
+
+    // ASSERT
+    assertNotNull(form);
+    assertFalse(form.isEmpty());
+    assertEquals("NCIt Term Suggestion Request", form.get("formName").asText());
+    // TODO: Update this to ncithesaurus@mail.nih.gov after the form is updated
+    assertEquals("agarcia@westcoastinformatics.com", form.get("recipientEmail").asText());
+  }
+
+  /**
+   * Integration test for submitting a filled out form and sending the email. NOTE: Set your local
+   * environment variables in your config. Your test email will need an App Password for access, if
+   * using Gmail.
+   *
+   * @throws Exception exception
+   */
+  @Test
+  public void integrationTestSubmitForm() throws Exception {
+    // SET UP
+    final String formPath = "formSamples/submissionFormTest.json";
+    final JsonNode formData = createForm(formPath);
+    final String requestBody = objectMapper.writeValueAsString(formData);
+
+    // Mock the RecaptchaService to always return true for verifyRecaptcha
+    when(captchaService.verifyRecaptcha(anyString())).thenReturn(true);
+
+    // ACT & ASSERT - Verify the email was sent to the receiver in the form
+    log.info("Form data = {}", formData);
+    final MvcResult result =
+        this.mvc
+            .perform(
+                MockMvcRequestBuilders.post(baseUrl)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .header("Captcha-Token", recaptchaToken)
+                    .content(requestBody))
+            .andExpect(status().isOk())
+            .andReturn();
+  }
+
   /**
    * Helper method for creating a JsonNode from a Json file.
```

Comment:
Should fix the boilerplate comments here to the new name/purpose of the function, for future reference

Reply:
It's not a boiler plate comment, I wrote that. Should it say something else, because I'm creating a JsonNode from a jason file to create the form.

Reply:
Ok that should be fine then.